dev/io.openliberty.microprofile.telemetry.1.1.internal/original.bnd

```diff
@@ -0,0 +1,21 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+
+Bundle-Name: io.openliberty.microprofile.telemetry.1.1.internal
+Bundle-SymbolicName: io.openliberty.microprofile.telemetry.1.1.internal
+
+Export-Package: \
+	io.openliberty.microprofile.telemetry11.internal.cdi;version='1.0',\
+	io.openliberty.microprofile.telemetry11.internal.rest;version='1.0';thread-context=true
```

Comment:
Why do we have these exports in the original.bnd, rather than in the bnd.bnd?

Usually the original and transformed versions of a bundle export the same packages.